Mediator-Error_Handling.md

```diff
@@ -31,28 +31,28 @@ This table defines the expected the behavior of the mediator in different scenar
 
 ### Scenarios Description
 
-#### Genetal Scenarios:
+#### General Scenarios:
 
 - **G1** - Received the exact same message (encrypted with the same ephemeral keys) (possible replay attack)
   - S1A - (default) executes the protocol as normal
   - S1B - ignore (no effect/output) 
   - S1C - Send a problem report "e.p.cryto.dejavu" ??? or "e.p.crypto.replay"
   - S1D - Reply with the exact same message (encrypted with the same ephemeral keys)
 
-- **G2** - Receive a message for a unsupported protocol
+- **G2** - Receive a message for an unsupported protocol
   https://input-output.atlassian.net/browse/ATL-5840
   - G2A - ignore (no effect/output) 
   - G2B - Send a problem report "e.p.msg.unsupported"
 
-- **G3** - Receive a message for a unsupported version of the protocol.
+- **G3** - Receive a message for an unsupported  unsupported version of the protocol.
```

Comment:
- Remove the second "unsupported"

Reply:
Almost lost your review if not for the email.
I will have a new PR with it